### PR TITLE
Refined the article detail page and added extra components

### PR DESCRIPTION
**Pull request by AkashGutha**
giphy-downsized 2

Added some comments, so that I can keep track of the sections in the page.
I'll remove them once the whole page is done.

### CODE REVIEW

**Comment by AkashGutha** [comment]
screencapture-localhost-3000-1507766920340

**Comment by AkashGutha** [comment]
This is where we are now :)

**Comment by AkashGutha** [comment]
screencapture-localhost-3000-1507948670261

Completed the sharing UI.

**Comment by AkashGutha** [comment]
This is feature complete.
Working as a normal blog would.

**Comment by AkashGutha** [comment]
The pages are connected now.
it's complete. now. leaving some rough edges.

File: src/Pages/LandingPage.js [hunk 1]
```diff
@@ -48,11 +59,50 @@ const LandingPageObserver = observer(
           >
             {articles.map((article, index) => {
               if (index === 3) {
-                return <LargeCardPost article={article} />;
+                return (
+                  <div
```
Comment (joelhooks) [inline src/Pages/LandingPage.js:63]: Should these just be React Router `Link`s? It's not significant at the moment, just curious.

**Review by joelhooks — APPROVED** [review]

File: src/Pages/LandingPage.js (hunk 1 above)
Comment (AkashGutha) [reply src/Pages/LandingPage.js:63]: The Link element is causing the text inside the card to be converted to hyperlinks.
We can use CSS.
I chose to use programmatic navigation, hence the onCLick events.

**Pull request merged**